### PR TITLE
remove archived c-lightning-plugin-csvexportpays

### PR DESCRIPTION
**Pull request by daywalker90**
@0xB10C This plugin is archived and has no tests. It should therefore be removed.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -46,3 +43,6 @@
 [submodule "watchtower-client"]
 	path = watchtower-client
 	url = https://github.com/talaia-labs/rust-teos
+[submodule "cln-ntfy"]
```
Comment (chrisguida) [inline .gitmodules:46]: What is this submodule doing here?

**Comment by daywalker90** [comment]
I removed that cln-ntfy entry in this PR

**Comment by chrisguida** [comment]
ACK 52aa82cfdab2c9cc5ebe2084351e232e1771b435

**Comment by chrisguida** [comment]
@mergifyio queue

**Comment by chrisguida** [comment]
@mergifyio refresh

**Comment by chrisguida** [comment]
@mergifyio queue

**Review by chrisguida — APPROVED** [review]
Testing to see if this approval auto-merges

**Pull request merged**